crates/bevy_reflect/bevy_reflect_derive/src/utility.rs

```diff
@@ -29,6 +29,13 @@ pub(crate) struct ResultSifter<T> {
     errors: Option<syn::Error>,
 }
 
+pub(crate) fn field_ident_or_indexed(index: usize, ident: Option<&Ident>) -> Member {
+    ident.as_ref().map_or_else(
+        || Member::Unnamed(index.into()),
+        |&ident| Member::Named(ident.clone()),
+    )
```

Comment:
Nit: The `as_ref` is actually redundant here since we pass the ident by reference
```suggestion
    ident.map_or_else(
        || Member::Unnamed(index.into()),
        |ident| Member::Named(ident.clone()),
    )
```

Reply:
Oh dang, and I added that `&` to satisfy the compiler. Good catch!